content/200-orm/800-more/500-development-environment/050-environment-variables/040-env-files.mdx

```diff
@@ -31,7 +31,7 @@ We suggest to move the contents of prisma/.env to .env to consolidate your env v
 
 The following table describes where the Prisma CLI looks for the `.env` file:
 
-| **Command**                                     | **Schema file location**                                                     | **`.env` file locations checked, in order**               |
+| **Command**                                     | **schema location**                                                     | **`.env` file locations checked, in order**               |
```

Comment:
Unsure what we actually did to `.env` loading: https://prisma-company.slack.com/archives/C058VM009HT/p1717448728682249